### PR TITLE
Added test for PDF011

### PR DESCRIPTION
**Pull request by leetaiying**
Closing issue #13.

### CODE REVIEW

File: tests/namespace_usage_test.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        pytest.param(
+            "from pandas import Categorical\n"
+            "import pandas as pd\n"
+            "\n"
+            "cat1 = Categorical()\n"
+            "cat2 = pd.Categorical()\n",
+            "5:7: PDF011 found both 'pd.foo' and 'foo' in the same file",
+            id="One is accessed from pd, the other is imported from pandas",
+        ),
+    ),
```
Comment (MarcoGorelli) [inline tests/namespace_usage_test.py:48]: Awesome!

Can we add an extra test in which we have
```
import pandas
from pandas import Categorical

cat1 = Categorical()
cat2 = pd.Categorical()
```
?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: tests/namespace_usage_test.py (hunk 1 above)
Comment (MarcoGorelli) [reply tests/namespace_usage_test.py:48]: if it doesn't pass, that's OK, let me know and I'll open a new issue for it and we can merge this one as it is

Comment (leetaiying) [reply tests/namespace_usage_test.py]: Thanks! Just added it.

**Review by MarcoGorelli — APPROVED** [review]
Awesome, thank you @leetaiying !

**Comment by leetaiying** [comment]
Thank you @MarcoGorelli!!

**Pull request merged**